
let items;

try {
    items.forEach(item => {
        console.log(item);
    });   
} catch (error) {
    console.error("Error : ", error);
    /* items = [1, 2, 3];
    items.forEach(item => console.log(item)); */
}

console.log("Error Management is working successfully.");

let info = "We are coding.";

console.log(info);










